modules/pages/client/views/faq-general.client.view.html

```diff
@@ -106,16 +106,16 @@ <h3>How can I remove my account</h3>
 
 <div class="faq-question" id="how-can-i-help">
   <h3>How can I help?</h3>
-  Trustroots is a community of travellers for sharing, hosting and getting people together. If you want to help grow this network you can find some <a href="https://github.com/Trustroots/trustroots/wiki/Volunteering">things to do here</a>.
+  Trustroots is a community of travellers for sharing, hosting and getting people together. If you want to help grow this network you can find some <a href="/volunteering">things to do here</a>.
```

Comment:
We should have internal links in this form:

```html
<a ui-sref="volunteering"></a>
```

That way the application won't refresh the whole page between page transitions, they get logged in statistics as page views and changing the URL in one place will update it everywhere.

You can see different state codes relevant for this PR here: https://github.com/Trustroots/trustroots/blob/018cd6996f9bfe4ce390b2cce2562196edab3e59/modules/pages/client/config/pages.client.routes.js

...and for each module there's a similar client.routes.js file if you need to link e.g. profile pages etc.